sources/collections/tests/bit-vector-xor.dylan

```diff
@@ -40,72 +40,293 @@ define test bit-vector-xor-tiny-vector
+
+  let bits5 = random-elements($huge-size, seed: 189);
+  let not-bits5 = compute-not-bits(bits5, $huge-size);
+  let vector5 = make(<bit-vector>, size: $huge-size);
+  set-bits(vector5, bits5);
+
+  let bits6 = random-elements($huge-size, seed: 194);
+  let not-bits6 = compute-not-bits(bits6, $huge-size);
+  let vector6 = make(<bit-vector>, size: $huge-size, fill: 1);
+  unset-bits(vector6, not-bits6);
+
+  let bits5-xor-bits6
+    = intersection(union(bits5, bits6),
+                   compute-not-bits(intersection(bits5, bits6), $huge-size));
   let bits6-xor-bits5 = bits5-xor-bits6;
 
   // Xor two all-zero vectors together with different pad values
   //
   test-xor-for-all-pad-values("Xor two all-zero vectors",
-    vector1, vector2,
+    $huge-size, vector1, vector2,
     #"all-zeros", #"all-zeros", #"all-zeros", #"all-zeros");
 
   // Xor two all-one vectors together with different pad values
   //
   test-xor-for-all-pad-values("Xor two all-one vectors",
-    vector3, vector4,
+    $huge-size, vector3, vector4,
     #"all-zeros", #"all-zeros", #"all-zeros", #"all-zeros");
 
   // Xor an all-zero and all-one vector together with different pad values
   //
   test-xor-for-all-pad-values("Xor an all-one and all-zero vector",
-    vector1, vector3,
+    $huge-size, vector1, vector3,
     #"all-ones", #"all-ones", #"all-ones", #"all-ones");
   test-xor-for-all-pad-values("Xor an all-zero and all-one vector",
-    vector3, vector1,
+    $huge-size, vector3, vector1,
     #"all-ones", #"all-ones", #"all-ones", #"all-ones");
 
   // Xor an all-zero vector with a mixed vector and different pad values
   //
   test-xor-for-all-pad-values("Xor an all-zero and mixed vector",
-    vector1, vector5,
+    $huge-size, vector1, vector5,
     bits5, bits5, bits5, bits5);
   test-xor-for-all-pad-values("Xor a mixed and all-zero vector",
-    vector5, vector1,
+    $huge-size, vector5, vector1,
     bits5, bits5, bits5, bits5);
 
   // Xor an all-one vector with a mixed vector and different pad values
   //
   test-xor-for-all-pad-values("Xor an all-one and mixed vector",
-    vector3, vector5,
+    $huge-size, vector3, vector5,
       not-bits5, not-bits5, not-bits5, not-bits5);
   test-xor-for-all-pad-values("Xor a mixed and all-one vector",
-    vector5, vector3,
+    $huge-size, vector5, vector3,
       not-bits5, not-bits5, not-bits5, not-bits5);
 
   // Xor two mixed vectors together with different pad values
   //
   test-xor-for-all-pad-values("Xor two vectors",
-    vector5, vector6,
+    $huge-size, vector5, vector6,
     bits5-xor-bits6, bits5-xor-bits6, bits5-xor-bits6, bits5-xor-bits6);
   test-xor-for-all-pad-values("Xor two vectors again, arguments reversed",
-    vector6, vector5,
+    $huge-size, vector6, vector5,
     bits6-xor-bits5, bits6-xor-bits5, bits6-xor-bits5, bits6-xor-bits5);
 
 end test;
 
+define test bit-vector-xor-multiple-word-vector
+    (description: "Test bit-vector-andc2 with multiple-word sized bit-vectors")
```

Comment:
Testworks no longer supports the "description" attribute, so that can be removed here. Just use a comment. The description slot was never used for anything.

(OD doesn't warn for extra keywords passed to `make`, which is what this ends up doing.)

I suggest moving all the descriptions to comments in a separate commit. Doesn't have to be in this PR either.

Reply:
Should I be replacing `check-*` with `assert-*` too?

Reply:
Not necessarily.  The plan is to change the semantics of `assert-*` to abort the rest of the test if they fail, whereas `check-*` will retain the current behavior.  If you want the assertion to abort the test you could switch, but I'd leave it for a different PR.

Given this change of plans (i.e, when I introduced `assert-*` it was purely to have a replacement for `check-*` that could auto-generate the failure message based on the code) I'm not sure what to do about the fact that `assert-*` and `check-*` have different parameter lists.